localstack-core/localstack/services/sns/publisher.py

```diff
@@ -569,13 +569,16 @@ def _publish(self, context: SnsPublishContext, subscriber: SnsSubscription):
         region = extract_region_from_arn(subscriber["Endpoint"])
         ses_client = connect_to(aws_access_key_id=account_id, region_name=region).ses
         if endpoint := subscriber.get("Endpoint"):
+            # TODO: legacy value, replace by a more sane value in the future
+            #  no-reply@sns-localstack.cloud or similar
+            sender = config.SNS_SES_SENDER_ADDRESS or "admin@localstack.com"
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches and the blocking or non-blocking decorations to further communicate your intent.  -->
**suggestion:** ‏Would be great to have a "localstack only" test for this newly introduced config value.

Reply:
Very good point, I've added the test now, tell me what you think 😄